src/main/java/nl/novi/techiteasy1121/controllers/CIModuleController.java

```diff
@@ -0,0 +1,58 @@
+package nl.novi.techiteasy1121.controllers;
+
+
+import nl.novi.techiteasy1121.dtos.CIModuleDto;
+import nl.novi.techiteasy1121.services.CIModuleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+// Dit is de Controller klasse van de CIModule entiteit en heeft vergelijkbare methodes als de TelevisionController
```

Comment:
Zorg ervoor dat je de comments in de code up-to-date houdt en dat ze relevant zijn voor de huidige implementatie.